project/tests/test_uploading_images.py

```diff
@@ -0,0 +1,74 @@
+import shutil
+import pathlib
+from config import BASEDIR
+from project.models import UploadedImage
+from project.tasks.uploads import celery_make_thumbnail as make_thumbnail
+from unittest.mock import patch
+from werkzeug.datastructures import FileStorage, Headers
+from project.tests.utils import ProjectTestCase
+
+
+class TestUploadImage(ProjectTestCase):
+
+    def setUp(self):
+        self.upload_folder = pathlib.Path(self.app.config['UPLOAD_FOLDER'])
```

Comment:
[tempfile — Generate temporary files and directories](https://docs.python.org/3.4/library/tempfile.html)